spec/integration/cli/server_spec.rb

```diff
@@ -105,6 +105,23 @@ def call(params)
         end
       end
     end
+
+    context "when not enabled" do
+      it "does not log request" do
+        with_project(project) do
+          touch log
+          replace "config/environment.rb", "logger level: :debug", "logger stream: false"
+
+          server do
+            visit "/"
+            expect(page).to have_title("Hanami | The web, with simplicity")
+          end
+
+          content = contents(log)
+          expect(content).to be_empty
```

Comment:
That's because is `be_an_existing_file`, not `be_existing_file`. Sorry about the confusion.